src/main/java/com/smartbear/readyapi/client/extractors/Extractors.java

```diff
@@ -0,0 +1,42 @@
+package com.smartbear.readyapi.client.extractors;
+
+import com.smartbear.readyapi.client.teststeps.propertytransfer.PathLanguage;
+
+public class Extractors {
+
+    /**
+     * Extracts a property from the teststep it was added on with the function "withProperties". The property must exist
+     * on the given teststep. Runs the lambda operation given in the ExtractorOperation field after the testserver run
+     *
+     * @param property The property to be extracted
+     * @param operator The lambda function to be run on the extracted property
+     * @return The extractor that is added to the testserver run
+     */
+    public static Extractor extractorProperty(String property, ExtractorOperator operator) {
```

Comment:
perhaps this should be named propertyExtractor - and the below method pathExtractor ?